packages/cli/README-source.md

```diff
@@ -1215,7 +1219,11 @@ have incomprehensible messages for end users, or possibly go uncaught.
 
 Zapier provides a couple tools to help with error handling. First is the `afterResponse`
 middleware ([docs](#using-http-middleware)), which provides a hook for processing
-all responses from HTTP calls. The other tool is the collection of errors in
+all responses from HTTP calls. Second is `response.throwForStatus()`
+([docs](#http-response-object)), which throws an error if the response status
+indicates an error. Since v10, `afterResponse` defaults to calling `throwForStatus`
+and using `afterResponse` makes you responsible for error response handling
+([docs](#error-response-handling)). The last tool is the collection of errors in
```

Comment:
Updated the general error handling topic to explain the role of the default `afterResponse` and how using it makes you responsible.